plenum/server/node.py

```diff
@@ -2533,11 +2533,15 @@ def checkPerformance(self) -> Optional[bool]:
             if backup_throughput is not None:
                 self.metrics.add_event(MetricsName.MONITOR_AVG_THROUGHPUT, backup_throughput)
 
-            master_latencies = self.monitor.getAvgLatency(self.instances.masterId).values()
+            master_latencies = self.monitor.getLatencies(self.instances.masterId).values()
             if len(master_latencies) > 0:
                 self.metrics.add_event(MetricsName.MASTER_MONITOR_AVG_LATENCY, mean(master_latencies))
 
-            backup_latencies = self.monitor.getAvgLatency(*self.instances.backupIds).values()
+            backup_latencies = {}
+            for lat_item in [self.monitor.getLatencies(instId) for instId in self.instances.backupIds]:
+                for cid, lat in lat_item.items():
+                    backup_latencies.setdefault(cid, []).append(lat)
+            backup_latencies = [mean(lat) for cid, lat in backup_latencies.items()]
```

Comment:
We show mean value here (for all clients), while monitor checks it for each client independently.